internal/msg/errormsg.go

```diff
@@ -221,3 +221,11 @@ const (
 	// InsightsReportError indicates failure push to insights.
 	InsightsReportError = "unable to report result to insights"
 )
+
+// npm related messages
+const (
+	// NpmEmptyScope indicates a missing scope value
+	NpmEmptyScope = "scope of scoped registry %d is empty"
+	// NpmEmptyURL indicates a missing URL value
+	NpmEmptyURL = "url of scoped registry %d is empty"
```

Comment:
Just a bit more context for what the number refers to:

```suggestion
	NpmEmptyScope = "scope of scoped registry at index %d is empty"
	// NpmEmptyURL indicates a missing URL value
	NpmEmptyURL = "url of scoped registry at index %d is empty"
```